src/utils.rs

```diff
@@ -0,0 +1,16 @@
+pub mod utils {
+    use std::env;
+    use std::fs;
+
+    pub fn is_program_in_path(program: &str) -> bool {
+        if let Ok(path) = env::var("PATH") {
+            for p in path.split(":") {
+                let p_str = format!("{}/{}", p, program);
```

Comment:
it might not produce the expected result if program is already an absolute path

Reply:
Will look into this, thanks for reviewing this btw! I appreciate it :)